Aliucord/src/main/java/com/aliucord/settings/Crashes.java

```diff
@@ -64,14 +66,63 @@ public Context getContext() {
     @SuppressLint("SetTextI18n")
     public void onViewBound(View view) {
         super.onViewBound(view);
-
         setActionBarTitle("Crash Logs");
 
         Context context = requireContext();
-        LinearLayout layout = (LinearLayout) ((NestedScrollView) ((CoordinatorLayout) view).getChildAt(1)).getChildAt(0);
+        int padding = Utils.getDefaultPadding();
+        int p = padding / 2;
+        File folder = new File(Constants.BASE_PATH, "crashlogs");
+        File[] files = folder.listFiles();
+        assert files != null;
+
+        AppCompatImageButton crashFolderBtn = new AppCompatImageButton(context);
+        int ic1 = R$d.ic_open_in_new_white_24dp;
```

Comment:
why not put these directly into the getDrawable? extracting them makes it less readable